src/Surfnet/Stepup/Identity/Event/GssfPossessionProvenAndVerifiedEvent.php

```diff
@@ -30,10 +30,20 @@
 use Surfnet\Stepup\Identity\Value\StepupProvider;
 use Surfnet\StepupBundle\Value\SecondFactorType;
 use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\Forgettable;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\RightToObtainDataInterface;
 use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\SensitiveData;
 
-class GssfPossessionProvenAndVerifiedEvent extends IdentityEvent implements Forgettable, PossessionProvenAndVerified
+class GssfPossessionProvenAndVerifiedEvent extends IdentityEvent implements Forgettable, PossessionProvenAndVerified, RightToObtainDataInterface
 {
+    protected static $whitelist = [
+        'identity_id',
+        'identity_institution',
+        'second_factor_id',
+        'stepup_provider',
+        'registration_requested_at',
+        'preferred_locale',
+    ];
```

Comment:
Why are the: SF Identifier, Common name and Email not on the whitelist?

Reply:
added  'gssfId', 'commonName', 'email',

Reply:
added